README.md

```diff
@@ -3,6 +3,10 @@
 
 # Cloud SQL Modules
 
+<!-- NOTE: We use absolute linking here instead of relative linking, because the terraform registry does not support
+           relative linking correctly.
+-->
+
```

Comment:
Missed links:
- `cloud-sql` module reference in the section listing out the "primary modules"
- `Gruntwork Philosophy` link
- `Contribution Guidelines` link
- `LICENSE` link
- `Releases` link

Reply:
Ahh, love the IDEs... I seemed to have some magical IntelliJ filter in my link search. Will fix!